src/common.speech/HeaderNames.ts

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+export class HeaderNames {
+    public static AuthKey: string = "Ocp-Apim-Subscription-Key";
+    public static ConnectionId: string = "X-ConnectionId";
+    public static ContentType: string = "Content-Type";
+    public static CustomCommandsAppId: string = "X-CommandsAppId";
+    public static Path: string = "Path";
+    public static RequestId: string = "X-RequestId";
+    public static RequestStreamId: string = "x-streamid";
```

Comment:
Should this be X-StreamId?

Reply:
This was a direct copy and paste, but a search in the services code (`HeaderConstants.cs` for inquiring minds) reveals that it is the `"X-StreamId"` we'd expect there, which should make it safe to update. I'll do that. Thanks for drawing attention to it, too -- it bothered me but I didn't think to follow up!